src/Fixer/Import/GroupImportFixer.php

```diff
@@ -23,12 +27,19 @@
 use PhpCsFixer\Tokenizer\CT;
 use PhpCsFixer\Tokenizer\Token;
 use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Utils;
+use Symfony\Component\OptionsResolver\Exception\InvalidOptionsException;
 
 /**
  * @author Volodymyr Kupriienko <vldmr.kuprienko@gmail.com>
+ * @author Greg Korba <greg@codito.dev>
  */
-final class GroupImportFixer extends AbstractFixer
+final class GroupImportFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
+    public const GROUP_CLASS_LIKES = 'class-likes';
```

Comment:
`private`? I see they are used in tests, but I'd use the strings directly in testd.

At least `@internal`, please.

Reply:
I have mixed feelings about it, as I believe it's good to provide a contract for options and values, so people can use it in their configs and feel safe about the behaviour. I think we already merged some fixers with public constants.

Reply:
> I believe it's good to provide a contract for options and values

I'm not against it, but if/when we do it, let's do it properly, e.g. have the constant in docs.

Reply:
You mean in this PR, or in general?

Reply:
In genera, of course.

Reply:
After some offline discussion we decided to mark these constants as internal, and try to refactor current codebase to extract new public contract that would provide these 3 import types for usage in any place of the code. Also, as it was pointed out, we currently have "classy" naming convention for class-like stuff (classes, interfaces, traits, enums) that are covered by standard `use` import, that's why one of the constants introduced in this PR was renamed (also with proper value).